src/amo/components/SponsoredAddonsShelf/index.js

```diff
@@ -23,6 +24,9 @@ import type { DispatchFunc } from 'core/types/redux';
 
 import './styles.scss';
 
+export const EVENT_URL = `${API_BASE}${config.get(
```

Comment:
Why is this needed?

Reply:
I guess I was just trying to reuse logic we already have for calculating an API URL. I can change it to 
```
`${config.get('apiPath')}${config.get('apiVersion')}/shelves/sponsored/event/`
```

to keep it relative.

Reply:
FTR, I have updated the code to use the above.